JDLApplicationOptions.md

```diff
@@ -35,3 +35,4 @@
 | useSass              | BOOLEAN  | Boolean               |                                                                                      | true or false                                                                                                                 |
 | authenticationType   | STRING   | EnumLiteral           | jwt, session, oauth2                                                                   | One of the listed values                                                                                                      |
 | clientTheme          | STRING   | AnyLiteral            | none                                                                                 | You can put whatever value you want,  provided you know it will work (like yeti)                                               |
+| rememberMeKey        | STRING   | String                |                                                                                      | Remember me key                                                                                                               |
```

Comment:
not sure it is used any more

Reply:
...hm, it looks like it is used [here](https://github.dev/jhipster/generator-jhipster/blob/00bf9acec6f9ee905e3559f37d40a2288b24af27/test/integration-test-reproducibility.spec.js#L47-L47). I think it would be more user-friendly if we can put there a better description (or a link) to explain the purpose of this option (which would also appear as hover text in the ide)

Reply:
...and [here](https://github.com/jhipster/generator-jhipster/blob/00bf9acec6f9ee905e3559f37d40a2288b24af27/test/jdl/test-files/applications_with_and_without_entities.jdl#L6) is also a test file